fork-visualizer/README.md

```diff
@@ -0,0 +1,36 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+
+## Getting Started
+
+First, run the development server:
+
+```bash
+npm run dev
+# or
+yarn dev
+# or
+pnpm dev
+# or
+bun dev
+```
```

Comment:
Yeah, I agree that we should choose one official one and stick with it and support only that one. And then commit the lock file. I'm fine with either npm or yarn because I've used both and know they are both popular. I've never heard of pnpm or bun and would prefer to keep it that way :)